pipeline.py

```diff
@@ -48,19 +59,32 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        pathlib_path_to_data = Path(self.path_to_raw_txt_data)
+
+        for file_path in pathlib_path_to_data.iterdir():
```

Comment:
you should recall the way to find all the files by a givem template for a name rather than making conditional logic with a global constant